tests/units.ua

```diff
@@ -211,6 +211,8 @@
 ⍤⤙≍ [2 10 5] ⬚5⋕{"2" "10" "dog"}
 ⍤⤙≍ [ℂ 3.4e6 1.2 ℂ π ∞] ⋕ {"1.2r3.4e6i" "infrpii"}
 ⍤⟜≍ ⋕°⋕. [ℂ 3.4e6 1.2 ℂ π ∞]
+⍤⤙≍ [ℂ4.5 `12.3 ℂ4.5 12.3 ℂ`4.5 12.3 ℂ`4.5 12.3 ℂ4.5 12.3] ◇⋕ {"-12.3r4.5i" "12.3r4.5i" "12.3-4.5i" "12.3r-4.5i" "12.3r4.5i"}
```

Comment:
Why `◇ content`?

Can you add some tests to verify that integers also work?
And `pi`, `inf` etc.